packages/prompts/src/index.ts

```diff
@@ -38,6 +38,11 @@ const S_CORNER_TOP_RIGHT = s('╮', '+');
 const S_CONNECT_LEFT = s('├', '+');
 const S_CORNER_BOTTOM_RIGHT = s('╯', '+');
 
+const S_INFO = s('ℹ', 'i');
+const S_SUCCESS = s('✔', '√');
+const S_WARN = s('⚠', '!');
+const S_ERROR = s('✘', 'x');
```

Comment:
```suggestion
const S_INFO = s('●', '•');
const S_SUCCESS = s('◆', '*');
const S_WARN = s('▲', '!');
const S_ERROR = s('■', 'x');
```

I don't mind the traditional symbols but I think the geometric shapes align with the prompt style a bit better.

<img width="649" alt="CleanShot 2023-02-24 at 17 03 28@2x" src="https://user-images.githubusercontent.com/7118177/221317296-a546862d-5983-473f-84a7-67036070a8e9.png">

Reply:
Man, how I would kill for your designer's eye 😍